HLTrigger/Egamma/plugins/HLTDisplacedEgammaFilter.cc

```diff
@@ -61,6 +64,7 @@ void HLTDisplacedEgammaFilter::fillDescriptions(edm::ConfigurationDescriptions&
   desc.add<double>("sMaj_max", 999.0);
   desc.add<double>("seedTimeMin", -25.0);
   desc.add<double>("seedTimeMax", 25.0);
+  desc.add<bool>("useTrackVeto", false);
```

Comment:
```suggestion
  desc.add<bool>("useTrackVeto", true);
```
The default should be `true`, such that the the outputs are unmodified by this PR. If certain modules in the HLT configs will need to use `useTrackVeto=false`, such change needs to be requested via a JIRA ticket.